Emergence/Contexts/Networking/ArtsyAPI.swift

```diff
@@ -77,12 +76,28 @@ extension ArtsyAPI : MoyaTarget {
         case .ShowInfo(let showID):
             return "/api/v1/show/\(showID)"
 
-        case .ClosingShowsNearLocation(_, _), .UpcomingShowsNearLocation(_, _):
+        case .RunningShowsNearLocation(_, _, _), .UpcomingShowsNearLocation(_, _), .PastShowsNearLocation(_, _):
```

Comment:
If you don't care about any of the extracted values, you can omit them altogether:

```swift
case .RunningShowsNearLocation, .UpcomingShowsNearLocation, .PastShowsNearLocation:
```